OptimizelyDemoApp/AppDelegate.swift

```diff
@@ -125,8 +125,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
             })
 #endif
-            let variation = optimizelyClient?.activate((self?.experimentKey)!, userId: (self?.userId)!, attributes: (self?.attributes))
+            let variation = optimizelyClient?.activate((self?.experimentKey)!, userId: (self?.userId)!)
             
+            if let experiments = optimizelyClient?.optimizely?.config?.experiments {
+                for experiment in experiments {
+                    print(experiment.experimentKey)
```

Comment:
Print? Is this desired?

Reply:
it's the demo app.  this is just proving that the change works.